core/chains/evm/txmgr/txmgr.go

```diff
@@ -53,6 +53,17 @@ type TxManager[
 	Reset(f func(), addr ADDR, abandon bool) error
 }
 
+type TestTxManager[
+	CHAIN_ID txmgrtypes.ID,
+	HEAD txmgrtypes.Head,
+	ADDR types.Hashable,
+	TX_HASH types.Hashable,
+	BLOCK_HASH types.Hashable,
+] interface {
+	TxManager[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH]
+	Abandon(addr ADDR) (err error)
+}
```

Comment:
Unused?
```suggestion
```

Reply:
Thought it was going to be more useful... removed that and add the helper function instead.